### PR TITLE
statement on FSF reappointing Stallman to the board of directors

### PR DESCRIPTION
**Pull request by ywwg**

### CODE REVIEW

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
title: "A Statement on Richard Stallman and the Free Software Foundation"
```

Just a suggestion, I don't mind much either way

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md]: Moreover, Mixxx is and always has been licensed under the terms of the GNU General Public License, version 2 *or later*, and only the Free Software Foundation can publish new versions of the GNU GPL which could be retroactively applied to the Mixxx code.
Therefore the Mixxx project has an interest in ensuring that the Free Software Foundation has inclusive, principled, and wise leadership which can effectively lead the broad free software community to meet present and future challenges.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md:17]: Furthermore, we call upon the Free Software Foundation to revise its bylaws to institute democratic processes for choosing leadership.
We believe that the myriad of software developers who chose to trust the FSF with stewardship of licenses for the code we wrote deserve to have input over the leadership of the FSF.
We cannot trust an organization that knowingly makes very controversial decisions behind closed doors.

**Comment by Be-ing** [comment]
Thank you for working on this.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md:2]: Maybe we should sign this as "The Mixxx Team"?

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Holzhaus) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
His departure in 2019 was a consequence of the criticism he has faced regarding his inappropriate behavior towards women, his hurtful comments about people with Down's Syndrome and his remarks regarding the Epstein scandal. 
His resignation came as a relief for many who have suffered his bad behavior.
We as free software developers strive build an inclusive and welcoming community and all of these points disqualify Richard Stallman to represent the FSF and the free software community as a whole.
```

File: content/news/2021-XX-XX-fsf-statement.md
Comment (ywwg) [reply content/news/2021-XX-XX-fsf-statement.md:2]: yes definitely

File: content/news/2021-XX-XX-fsf-statement.md
Comment (ywwg) [reply content/news/2021-XX-XX-fsf-statement.md]: added (slightly tweaked)

File: content/news/2021-XX-XX-fsf-statement.md
Comment (ywwg) [reply content/news/2021-XX-XX-fsf-statement.md:17]: added (slightly tweaked)

File: content/news/2021-XX-XX-fsf-statement.md
Comment (ywwg) [reply content/news/2021-XX-XX-fsf-statement.md]: done (slightly tweaked)

**Comment by ywwg** [comment]
pushed new version.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md]: This is redundant with the title of the blog post. I think it should be removed.
image

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md:13]: add paragraph break after this line

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
The FOSS community has led the way for the rest of the wider tech industry, showing that open standards and practices result in better products while respecting users' freedom, privacy, and community.
```

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
Our community needs to demonstrate a strong sense of accountability at this time.
We urge that the FSF board to do the right thing by accepting responsibility for this bad decision and enact specific commitments for change to rebuild trust.
```

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
We also call upon the Free Software Foundation to revise its bylaws to institute democratic processes for choosing leadership and increase the transparency of its operations.
```

File: content/news/2021-XX-XX-fsf-statement.md
Comment (daschuer) [inline content/news/2021-XX-XX-fsf-statement.md:9]:
```suggestion
```
We can't prove this statement and it should be clear anyway from the sentences before.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (daschuer) [inline content/news/2021-XX-XX-fsf-statement.md]: I think this blog post should be written in the sense of distancing from the FSF and not for calling resignation. For my feeling this was consense among the team. Else we could have sign the open letter. 
```suggestion
```

**Review by daschuer — CHANGES_REQUESTED** [review]

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md:9]: Whether someone is qualified for something is an opinion, not a fact. We can voice our opinion here.

Comment (daschuer) [reply content/news/2021-XX-XX-fsf-statement.md:9]: Yes, but we have decided to do a blog post and not sign as org, to take the chance to explain the situation regarding to Mixxx like other orgs did.

It does not work well to express an opinion when you speak for a whole group.

Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md:9]: I don't follow your reasoning. You seem to imply that you agree with the sentence but we shouldn't publish it unless we all agree? But don't we all agree on this?

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md]: I think it would be negligent to not make calls for specific actions that the FSF could be held accountable for.

Comment (daschuer) [reply content/news/2021-XX-XX-fsf-statement.md]: Or blog ist mit the place to communicate calls to the FSF. 
In addition it is pretentious to call such things from things based on hearsay.
I don't want to or part of a public storm, fueled by comments on social media.

This public rush it a strong evidence that the reinstallation of Stallman is read as movement away from inclusion.
Our business is to distancing us from that. 

Calls to the FSF need to be addressed to them.

Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md]: > Calls to the FSF need to be addressed to them.

Private communications are trivial for them to ignore. They need to be held accountable publicly or nothing will change.

Comment (daschuer) [reply content/news/2021-XX-XX-fsf-statement.md]: No other org has repeated this call on their webside.

Comment (Holzhaus) [reply content/news/2021-XX-XX-fsf-statement.md]: I have no idea how the FSF's decision to reinstate RMS came to be. I think their decision is wrong, but demanding the whole board of directors to step down - including the female board members who had to put up with Stallman's behavior - is something I don't necessarily agree with.

Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md]: > including the female board members who had to put up with Stallman's behavior

That's beside the point. She already stepped down.

Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md]: Would you agree to calling for Stallman's resignation alone?

Comment (daschuer) [reply content/news/2021-XX-XX-fsf-statement.md]: So you think the board members will read our call on the Mixxx webside and will act on it? 

The only thing this will happen if they actually read it is, they will become angry and will not treat us a serious partner. 

I think the FSF has already proposed future steps l, so this call is outdated any way.

Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md]: > I think the FSF has already proposed future steps l, so this call is outdated any way.

They have not committed to removing all the current board of directors.

Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md]: Doing nothing would be admitting defeat before even trying.

Comment (Holzhaus) [reply content/news/2021-XX-XX-fsf-statement.md]: > > I think the FSF has already proposed future steps l, so this call is outdated any way.
> 
> They have not committed to removing all the current board of directors.

Who is "she"? Both Kat Walsh and Odile Bénassy are listed as board members on the FSF's website.

https://www.fsf.org/about/staff-and-board

Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md]: Ah, sorry. Kat Walsh [has resigned](https://mastodon.social/@mindspillage/105948765536809476). I don't know about Odile Bénassy.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (ywwg) [reply content/news/2021-XX-XX-fsf-statement.md:9]: It's redundant to say "we believe" in a blog post about our beliefs. I can add it back but it's unnecessary verbiage because the existence of the blog post implies this is our belief as a group.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (ywwg) [reply content/news/2021-XX-XX-fsf-statement.md]: By allowing Stallman to return, the board of directors is tacitly approving of his behavior and saying it's excusable. I don't see how the board can remain as well. 

What about:
"Therefore, we feel an obligation to add our voices to the chorus of those who have expressed dismay at this turn of events and call for the resignation of Richard Stallman. The FSF board of directors, by allowing him to return, is giving the appearance of endorsing his conduct and it seems doubtful that they can remain effective without significant personnel changes and bylaw reforms."

Comment (ywwg) [reply content/news/2021-XX-XX-fsf-statement.md]: (committed something along those lines)

**Review by Be-ing — APPROVED** [review]

**Review by uklotzde — APPROVED** [review]
Thanks for keeping the part about the misconduct of RMS concise with only the necessary background information.

I like the reasoning why we as users of the GPL have a viable interest in the legitimization of the FSF as an organization.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
We urge the FSF board to do the right thing by accepting responsibility for this bad decision and enact specific commitments for change to rebuild trust.
```

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Holzhaus) [reply content/news/2021-XX-XX-fsf-statement.md:9]: It's redundant but it doesn't hurt either. If it helps so that we can agree on this, let's just add it.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Holzhaus) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
His resignation came as a relief for many who have suffered his actions.
```

Reading this again, we already used "behavior" in the sentence before and it reads a bit strange.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (daschuer) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
We acknowledge the Free Software Foundation's efforts to revise its bylaws to establish democratic processes for electing leadership and to increase the transparency of its activities.
```

Comment (daschuer) [reply content/news/2021-XX-XX-fsf-statement.md]: This reflects the current state better. We can't call something in a news addressing the Mixxx user.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (uklotzde) [reply content/news/2021-XX-XX-fsf-statement.md]: I also was not really happy with this sentence but didn't have any alternative at hand. Jan's proposal is perfect and omits the term "bad".

File: content/news/2021-XX-XX-fsf-statement.md
Comment (daschuer) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
We now expect the FSF board to take responsibility for this bad decision and making enact specific commitments for change to rebuild trust.
```
This way it matches to the current development. 

> We urge the

We can urge them in a letter addressed to them and we can tell our users about it.

**Comment by daschuer** [comment]
The post should reflect the current development. See my inline comments. 
The rest looks really good now. Thank you.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Holzhaus) [reply content/news/2021-XX-XX-fsf-statement.md]: I think "urge" fits better to be honest. And I think we can use it here, as it's basically kind of an open letter.

Comment (daschuer) [reply content/news/2021-XX-XX-fsf-statement.md]: Deepl translates this to: 

Wir fordern den FSF-Vorstand auf, das Richtige zu tun, indem er die Verantwortung für diese Fehlentscheidung übernimmt und konkrete Änderungszusagen macht, um das Vertrauen wiederherzustellen.

But we don't, we write a news blog. 

We could write: We have asked the FSF board ...
If we had done that, but so far we have not. 

Therefore I think it is correct to express our expectations here. In the sense of a last chance. 
Otherwise, we must clearly distance ourselves from the FSF, as Red Head has done. 
A contribution in this sense I would like more, but so it is also OK.

**Review by daschuer — APPROVED** [review]
I would prefere that my latest commenst would be integrated, But this should not block the release. Finally @ywwg  is the owner of this post.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Holzhaus) [reply content/news/2021-XX-XX-fsf-statement.md]: Even in German, "urge" ("auffordern") would be correct IMHO. We do it by publishing this article (like an open letter). "We expect" ("wir erwarten") sounds very antagonistic to me. But I don't have a hard opinion to this and leave it to @ywwg to decide.

Comment (daschuer) [reply content/news/2021-XX-XX-fsf-statement.md]: My idea was to acknowledge that they already have announced this step.

Comment (Holzhaus) [reply content/news/2021-XX-XX-fsf-statement.md]: Ok, maybe something like:
```suggestion
We expect the FSF board to do the right thing and follow up its statements with actions by accepting responsibility for this bad decision and enacting specific commitments for change to rebuild trust.
```

Comment (daschuer) [reply content/news/2021-XX-XX-fsf-statement.md]: Thank you. @ywwg The "expect" was meant like the angry mother expect when telling her son to tidy up. And not like expecting a snail post letter. Don't know if both versions exists in English.

Comment (Be-ing) [reply content/news/2021-XX-XX-fsf-statement.md]: That's good, just change "enact" to "enacting".

Comment (Holzhaus) [reply content/news/2021-XX-XX-fsf-statement.md]: Done.

**Comment by ywwg** [comment]
I integrated the latest changes, I'll take one more look and make sure everything has been incorporated.

In the future it might be better to use Dropbox Paper or Google Docs to write a post like this, github is extremely awkward for this use

**Comment by ywwg** [comment]
I added one more sentence to confirm that we don't just trust the board to change itself effectively.

File: content/news/2021-XX-XX-fsf-statement.md
Comment (Be-ing) [inline content/news/2021-XX-XX-fsf-statement.md]:
```suggestion
We do acknowledge the Free Software Foundation's efforts to revise its bylaws to establish democratic processes for electing leadership and to increase the transparency of its activities, and the community will be watching closely to verify that these changes are effective and substantial.
```

**Comment by Be-ing** [comment]
> In the future it might be better to use Dropbox Paper or Google Docs to write a post like this, github is extremely awkward for this use

I personally like using GitHub for collaborative document writing this way.

**Comment by Be-ing** [comment]
Thank you for writing the text here @ywwg.

**Pull request merged**
